script.js

```diff
@@ -1,4 +1,123 @@
+var player1Score = 0;
+var player2Score = 0;
+//total score for player
+var totalPlayer1Score = [];
+var totalPlayer2Score = [];
+var totalSumPlayer1 = 0;
+var totalSumPlayer2 = 0;
+//dices for players
+var player1Dice1 = 0;
```

Comment:
Just like how you stored totalPlayer1Score in an array, you could actually also store the player's dice in an array if you wanted. This will make your code scalable such that you could vary the number of dice used to play this game